.gitignore

```diff
@@ -1,9 +1,17 @@
+.DS_*
 *.log
-.DS_Store
-node_modules
+logs
+**/*.backup.*
+**/*.back.*
 .cache
-.rts2_cache_cjs
```

Comment:
Change the `.cache` to `*cache*` so it will ignore all of the Rollup typescript cache files.